src/main/kotlin/no/nav/personbruker/dittnav/eventaggregator/done/DoneBatchProcessor.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.personbruker.dittnav.eventaggregator.done
+
+import no.nav.personbruker.dittnav.eventaggregator.common.database.entity.Brukernotifikasjon
+import no.nav.personbruker.dittnav.eventaggregator.config.EventType
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class DoneBatchProcessor(private val existingEntitiesInDatabase: List<Brukernotifikasjon>) {
```

Comment:
Her kunne vi vurdert å droppet å bruke view-et, og heller hente inn en liste for hver eventtype. Da blir sjekken for å se hvilken type done-eventet tilhører veldig enkel, og det blir lettere å eventuelt støtte at eventer kun trenger å ha en unik ID innenfor en eventtype.

Reply:
Da kan jeg egentlig fikse dette med en gang, slik at vi kun looper gjennom aktive eventer. Vi har ferdige metoder for dette hvis man henter per eventtype.

Reply:
Da stemmer jeg for det.